src/app/auth/login-dialog/login-dialog.component.html

```diff
@@ -0,0 +1,32 @@
+@let m = mode();
+@let actionLabel = m === 'register' ? 'Register' : 'Login';
+<h2 mat-dialog-title>{{ actionLabel }}</h2>
+<mat-dialog-content>
+  @if (m === 'register') {
+    <p>Sign up with a new account.</p>
+  }
+<!-- FIXME enter on inputs (ngSubmit) not working -->
```

Comment:
oh see you already caught this

Reply:
ya, I'll try another way in next PR, it should just work with `(ngSubmit)`, but I think I could grab the event off the inputs directly.